content/en/docs/migration/helm-migration.md

```diff
@@ -0,0 +1,178 @@
+---
+title: "Migrate to the Helm Controller"
+linkTitle: "Migrate from the Helm Operator"
+description: "How to migrate from Helm Operator to Flux v2 and its Helm controller."
+weight: 30
+card:
+  name: migration
+  weight: 20
+---
+
+## Prerequisites
+
+- Completed initial migration with the Flux migration guide
+
+## Migration strategy
+
+Due to the high number of changes to the API spec, there are no detailed instructions available to provide a simple migration path. But there is a [simple procedure to follow](#steps), which combined with the detailed list of [API spec changes](#api-spec-changes) should make the migration path relatively easy.
+
+Here are some things to know:
+
+- The Helm Controller will ignore the old custom resources (and the Helm Operator will ignore the new resources).
+- Deleting a resource while the corresponding controller is running will result in the Helm release also being deleted.
+- Deleting a `CustomResourceDefinition` will also delete all custom resources of that kind.
+- If both the Helm Controller and Helm Operator are running, and both a new and old custom resources define a release, they will fight over the release.
+- The Helm Controller will always perform an upgrade the first time it encounters a new `HelmRelease` for an existing release; this is [due to the changes to release mechanics and bookkeeping](#helm-storage-drift-detection-no-longer-relies-on-dry-runs).
+
+The safest way to upgrade is to avoid deletions and fights by stopping the Helm Operator. Once the operator is not running, it is safe to deploy the Helm Controller (e.g., by following the [Get Started guide](../get-started/index.md), [utilizing `flux install`](../cmd/flux_install.md), or using the manifests from the [release page](https://github.com/fluxcd/helm-controller/releases)), and start replacing the old resources with new resources. You can keep the old resources around during this process, since the Helm Controller will ignore them.
+
+## Steps
+
+The recommended migration steps for a single `HelmRelease` are as follows:
+
+1. Ensure the Helm Operator is not running, as otherwise the Helm Controller and Helm Operator will fight over the release.
```

Comment:
```suggestion
1. Stop the Helm Operator.
1. Check the Helm Operator is not running.
{{% alert color="info" color="warning" title="'flux bootstrap' target" %}}
If the Helm Operator is still running, it will conflict with the Helm Controller when trying to deploy releases.
{{% /alert %}}
```